create_infotexts.py

```diff
@@ -195,6 +195,8 @@ def main():
         img = CypernImage()
         img.generate_list_of_stripped_keywords(metadata[fotonr]["Nyckelord"])
         img.create_commons_filename(metadata[fotonr])
+        img.add_cat_archaeological_exhibition_based_on_description(metadata[fotonr]["Beskrivning"])
```

Comment:
Thank you for clarification, that helps! I leave it as is, but I will also see what happened with the keywords mappings, which might affect the final implementation.